Fix one (hopefully latest) format string error. Thanks to Ludwig Nussel from
SUSE LINUX.
Index: xine-ui-0.99.4/src/xitk/main.c
===================================================================
--- xine-ui-0.99.4.orig/src/xitk/main.c
+++ xine-ui-0.99.4/src/xitk/main.c
@@ -1211,7 +1211,7 @@ static void event_listener(void *user_da
       }
       
       if(strlen(buffer))
-	report(buffer);
+	report("%s", buffer);
       
     }
     break;
